assets/js/explosion-effect.js

```diff
@@ -0,0 +1,56 @@
+const randomBetween = (min, max) => {
+  return Math.floor(Math.random() * (max + 1)) + min
+}
+
+const createExplosionElement = (x, y) => {
+  let element = document.createElement("div")
+
+  element.setAttribute("style", `left:${x - 150}px;top:${y - 150}px`)
+  element.setAttribute("class", "explosion")
+
+  return element
+}
+
+const createParticleElement = (x, y, color) => {
+  let element = document.createElement("div")
+
+  element.setAttribute("style", `background-color:rgb(${color});left:${x}px;top:${y}px`)
+  element.setAttribute("class", "particle")
+
+  return element
+}
+
+const explode = (x, y) => {
+  let particles = 15
+  let explosion = createExplosionElement(x, y)
+
+  document.body.appendChild(explosion)
+
+  for (let i = 0; i < particles; i++) {
+    let color = randomBetween(0, 255) + ',' + randomBetween(0, 255) + ',' + randomBetween(0, 255)
+    x = (explosion.clientWidth / 2) + randomBetween(80, 150) * Math.cos(2 * Math.PI * i / randomBetween(particles - 10, particles + 10))
```

Comment:
Maybe we can extract this bit to another function? Something like `createParticlePosition`? 🤔

Then it can return both positions at once in an object, something like:
```javascript
const {x, y} = createParticlePosition($ARGS)
```

Reply:
Also, what means these [magic numbers](https://www.eliotsykes.com/magic-numbers)? Ex. 1, 2, 10?

Could we move it to a variable that explains better what these numbers mean?